README.md

```diff
@@ -207,6 +207,10 @@ This will only be used if the size cannot be automatically calculated. This
 is currently only used and required if you supply a `Readable` stream as the
 file to upload. You may also use this to limit the position until which a file
 will be uploaded.
+* `agent = undefined`: specifies the [Agent](https://nodejs.org/api/http.html#http_class_http_agent)
```

Comment:
We usually use `null` as a default value for optional parameters. Please use it and also update the code.

Reply:
I accept your coding standard, although at this point I was trying to follow https://nodejs.org/dist/latest-v6.x/docs/api/http.html#http_http_request_options_callback convention, they use 'false', 'undefined', and Agent object, and 'null' is not an option.
I'm OK for having 'null' as default option, but it would probably need special handling in the code to pass 'undefined' to HTTP request. Please comment what you prefer.

Reply:
Done.